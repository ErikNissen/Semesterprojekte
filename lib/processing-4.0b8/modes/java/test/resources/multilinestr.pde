version https://git-lfs.github.com/spec/v1
oid sha256:1d91b63717c96d20cfc519ca2778b83d944c440003d525264e8404d4f69e230d
size 169
