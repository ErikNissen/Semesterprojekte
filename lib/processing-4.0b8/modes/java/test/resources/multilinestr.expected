version https://git-lfs.github.com/spec/v1
oid sha256:74c969df2aff054037c644efdb0fece51e9c634ceac7b5e1470ca5c4e04ceb0f
size 876
